ci-operator/step-registry/ipi/conf/vsphere/mirror-registry/ipi-conf-vsphere-mirror-registry-commands.sh

```diff
@@ -3,7 +3,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-mirror_registry_url="$(< /var/run/vault/vsphere/vmc_mirror_registry_url)"
+declare cloud_where_run
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+source "${SHARED_DIR}/vsphere_context.sh"
```

Comment:
The `vsphere_context.sh` is newly added into `CLUSTER_PROFILE` by dev, specific for this test case, right?

Reply:
no, it's generated by step [ipi-conf-vsphere-check](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh#L56) to record some platform information including dns server.